packages/react-slider/src/components/Slider/useSliderState.tsx

```diff
@@ -181,17 +215,17 @@ export const useSliderState = (state: SliderState) => {
       onKeyDownCallback?.(ev);
 
       if (ev.shiftKey) {
-        if (ev.key === 'ArrowDown' || ev.key === 'ArrowLeft') {
+        if (ev.key === 'ArrowDown' || ev.key === getRTLSafeKey('ArrowLeft')) {
           updatePosition(currentValue! - keyboardStep * 10, ev);
           return;
-        } else if (ev.key === 'ArrowUp' || ev.key === 'ArrowRight') {
+        } else if (ev.key === 'ArrowUp' || ev.key === getRTLSafeKey('ArrowRight')) {
```

Comment:
I don't think that it's a big deal, but `getRTLSafeKey()` will be executed 4 times in the worst case... It's not a big performance concern, but...

What about something like this?

```tsx
const normalizedKey = getRTLSafeKey(ev.key)

if (normalizedKey  === 'ArrowDown' || normalizedKey === 'ArrowLeft') {
```